packages/connect-voting-disputable/src/models/Vote.ts

```diff
@@ -48,11 +52,50 @@ export default class Vote {
     this.script = data.script
   }
 
+  get endDate(): string {
```

Comment:
Leaving a comment to point a good utility on the Connectors layer.

Reply:
It would be nice to have a better way to handle bignumber ops to be honest, a few helpers would be wonderful

Reply:
Wonder what you have in mind? Maybe make sense to have a `connect-utils`, @bpierre what you think?